src/kmq.ts

```diff
@@ -74,7 +74,7 @@ client.on("message", (message: Discord.Message) => {
 
 client.on("voiceStateUpdate", async (oldState, newState) => {
     let oldUserChannel = oldState.voiceChannel;
-    if (!oldUserChannel) {
+    if (!oldUserChannel || oldState.user !== client.user) {
```

Comment:
We still need to listen to user voice state changes for the the `User left voice channel, check if bot is only one left` case

Reply:
Thanks, can you take a look again? I made it how you originally suggested.